operator/deploy/olm-catalog/0.1.0/automationbrokeroperator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -41,13 +42,13 @@ spec:
       mediatype: image/png
   installModes:
   - type: OwnNamespace
-    supported: true
+    supported: false
```

Comment:
I'm curious what the OLM supported way of watching the `openshift-ansible-service-broker` namespace is.